cpu/cortexm_common/Makefile.features

```diff
@@ -2,6 +2,7 @@ FEATURES_PROVIDED += arch_32bit
 FEATURES_PROVIDED += arch_arm
 FEATURES_PROVIDED += arch_cortexm
 FEATURES_PROVIDED += periph_pm
+FEATURES_PROVIDED += cortexm_svc
```

Comment:
Please also add in `makefiles/features_modules.inc.mk`

```
# select cpu_check_address pseudomodule if the corresponding feature is used
USEMODULE += $(filter cortexm_svc, $(FEATURES_USED))
```

Reply:
done